### PR TITLE
Dtd 290

### PR DESCRIPTION
**Pull request by thadeusssewanyana**

### CODE REVIEW

**Review by beckfordp — APPROVED** [review]
👍

File: src/test/resources/features/sol/SolDebtDetailsUnhappyPath.feature [hunk 1]
```diff
@@ -24,15 +26,3 @@ Feature: Statement of liability Unhappy Path (Service Errors)
     When a debt statement of liability is requested
     Then the sol response code should be 404
     And the sol service will respond with {name: ReferenceDataLookupError, statusCode: 404, uniqueReference: 1085, 1000}
-
```
Comment (roccobruno) [inline src/test/resources/features/sol/SolDebtDetailsUnhappyPath.feature:27]: So u have not managed to fix this??

Comment (thadeusssewanyana) [reply src/test/resources/features/sol/SolDebtDetailsUnhappyPath.feature:27]: this didn't change, its as expected

File: src/test/resources/features/sol/SolDebtDetailsUnhappyPath.feature [hunk 2]
```diff
@@ -24,15 +26,3 @@ Feature: Statement of liability Unhappy Path (Service Errors)
...
-    Given debt details
-      | solType | debtId  | mainTrans | subTrans |
-      | UI      | debt006 | 1525      | 1000     |
-    And add debt item chargeIDs to the debt
-      | dutyId   |
-      | "duty01" |
-    When a debt statement of liability is requested
-    Then the sol response code should be 400
-    And the sol service will respond with {name: IfsDebtCalculationError, statusCode: 400, uniqueReference: }
-
```
Comment (roccobruno) [inline src/test/resources/features/sol/SolDebtDetailsUnhappyPath.feature:38]: but you have removed this test @thadeusssewanyana

**Pull request merged**